tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -1,38 +1,46 @@
 import React from 'react'
-import {
-  GridContainer,
-  Form,
-  Label,
-  TextInput,
-  Button,
-} from '@trussworks/react-uswds'
-
-import './EditProfile.scss'
-
-/**
- * This component renders when a user logs in for the first time
- * and needs to finish setting up their account. The user will
- * submit their first and last name along with their state, tribe or territory.
- */
 
 function EditProfile() {
   return (
-    <GridContainer>
+    <div className="grid-container">
       <h1 className="request-access-header font-serif-2xl">Request Access</h1>
       <p className="request-access-secondary">
         We need to collect some information before an OFA Admin can grant you
         access
       </p>
-      <Form>
-        <Label htmlFor="first-name">First name</Label>
-        <TextInput id="first-name" name="first-name" type="text" />
-        <Label htmlFor="last-name">Last name</Label>
-        <TextInput id="last-name" name="last-name" type="text" />
-        <Button className="usa-button--big" type="submit" disabled>
+      <form className="usa-form">
+        <label className="usa-label" htmlFor="first-name">
+          First name
+          <input
+            className="usa-input"
+            id="first-name"
+            name="first-name"
+            type="text"
+            required
+            aria-required="true"
+          />
+        </label>
+
+        <label className="usa-label" htmlFor="last-name">
+          Last name
+          <input
+            className="usa-input"
+            id="last-name"
+            name="last-name"
+            type="text"
+            required
+            aria-required="true"
+          />
+        </label>
+        <button
```

Comment:
It also seems like a `button` would make sense as a reusable component.

Reply:
Unfortunately, all of the buttons that I am using so far are completely different. I agree that pulling it out would be nice but I would prefer to save that work until I know which buttons will be reused regularly. I'm open to a conversation about it if you see other reasons.